src/Serilog/Context/LogContext.cs

```diff
@@ -94,8 +95,7 @@ public static IDisposable PushProperties(params ILogEventEnricher[] properties)
             var stack = GetOrCreateEnricherStack();
             var bookmark = new ContextStackBookmark(stack);
 
-            foreach (var prop in properties)
-                stack = stack.Push(prop);
+            stack = properties.Aggregate(stack, (current, prop) => current.Push(prop));
```

Comment:
Not sure this is idiomatic use of `Aggregate` here  - should be equivalent but maybe the original was clearer?